src/dialogs/examples/teams/UpdateCardMsgDialog.ts

```diff
@@ -59,7 +59,7 @@ export class UpdateCardMsgDialog extends TriggerActionDialog {
     ) {
         super(bot,
             DialogIds.UpdateCardMsgDialogId,
-            DialogMatches.Update_Card_Msg_Dialog_Intent,
+            DialogMatches.UpdateCardMsgDialogMatch,
```

Comment:
Please change back to Update_Card_Msg_Dialog_Intent, and use that constant as the text of the messageback button.

Reply:
Done.

Reply:
No, don't change Update_Card_Msg_Dialog_Intent to a regexp. This is only used to route the message; the user should never type the text herself, so there's no need for a regexp. Then you can use the string constant for the text of the messageBack action.